### PR TITLE
pyxirr 0.10.3

### PR DESCRIPTION
**Pull request by psteyer**
pyxirr 0.10.3 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-4749](https://anaconda.atlassian.net/browse/PKG-4749) 
- [Upstream repository](https://github.com/Anexen/pyxirr)

### Explanation of changes:

- initial recipe creation
- added smoke test using code snippet from upstream

[PKG-4749]: https://anaconda.atlassian.net/browse/PKG-4749?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+  host:
+    - python
+    - maturin >=1,<2
+    - pip
+    - setuptools
+    - wheel
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```
maturin is the build backend and is sufficient.

**Review by cbouss — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (psteyer) [reply recipe/meta.yaml]: Thank you! First time building a rust package. Is this normal when it comes to rust packages?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyxirr-{{ version }}.tar.gz
+  sha256: 8065865c327afef271dd5cd8561af22f407ccfde352b15860fe5aa0daee0af5d
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True #  [py37 or py>312]
```
Comment (ViridianMelody) [inline recipe/meta.yaml]:
```suggestion
  skip: True  # [py<37 or py>312]
```

**Review by ViridianMelody — APPROVED** [review]
Minor typo in skip block, otherwise looks good!

File: recipe/meta.yaml (hunk 1 above)
Comment (ViridianMelody) [reply recipe/meta.yaml]: Any package using another build backend such as `maturin` or `hatchling` wouldn't need `wheel`

File: recipe/meta.yaml (hunk 2 above)
Comment (psteyer) [reply recipe/meta.yaml]: Good catch, thank you!

**Pull request merged**